packages/embark-ens/src/index.js

```diff
@@ -373,16 +375,15 @@ class ENS {
       }
 
       this.events.request('code-generator:ready', () => {
-        this.events.request('code-generator:symlink:generate', location, 'eth-ens-namehash', (err, symlinkDest) => {
+        this.events.request('code-generator:symlink:generate', location, 'eth-ens-namehash', (err) => {
```

Comment:
You don't need to generate a symlink for namehash if you don't actually use it anymore